src/common/certificate-authorities/inject-system-cas.injectable.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+import { getInjectable } from "@ogre-tools/injectable";
+import { systemCAsInjectorInjectionToken } from "./system-cas-injector-token";
+
+const injectSystemCAsInjectable = getInjectable({
```

Comment:
Trivial: This level of abstraction is the correct place for responsibility of filtering ca's from wrong environments, not the ca-injectables itself (by producing empty arrays like now).

This is trivial here, but clear responsibilities of modules will carry us far in the future.